tests/launch.rs

```diff
@@ -2,13 +2,23 @@
 
 use kvm_ioctls::Kvm;
 
-use tdx::launch::TdxVm;
+use tdx::launch::{TdxVcpu, TdxVm};
+use tdx::tdvf;
 
 #[test]
 fn launch() {
     let kvm_fd = Kvm::new().unwrap();
+
+    // create vm
     let tdx_vm = TdxVm::new(&kvm_fd, 100).unwrap();
     let caps = tdx_vm.get_capabilities().unwrap();
     let _ = tdx_vm.init_vm(&kvm_fd, &caps).unwrap();
-    let _vcpufd = tdx_vm.fd.create_vcpu(0).unwrap();
+
+    // create vcpu
+    let vcpufd = tdx_vm.fd.create_vcpu(10).unwrap();
+    let tdx_vcpu = TdxVcpu::new(vcpufd).unwrap();
+    let mut firmware = std::fs::File::open("/usr/share/edk2/ovmf/OVMF.inteltdx.fd").unwrap();
```

Comment:
This is a path within the system you tested with, and there's no guarantee that another user would have it installed there.

Instead, create a `tests/data` directory and place this file there. You can then refer to it in this `File::open` operation.

Reply:
Another good point, thanks! Fixed.